core/domain/exp_services.py

```diff
@@ -1506,7 +1574,10 @@ def save_new_exploration(
                 'category': exploration.category,
             })])
     user_services.add_created_exploration_id(committer_id, exploration.id)
-    user_services.add_edited_exploration_id(committer_id, exploration.id)
+    user_services.add_edited_exploration_id_to_model(
```

Comment:
Again, this should follow the pattern (and be consistent for both created + edited exp ids):

```
    domain_object = get_from_model()
    domain_object.add_edited_exploration_id()
    save_domain_object_to_datastore()
```